brainstormforce/ultimate-addons-for-gutenberg
# Fix: FAQ deprecation when switch to `master to next-release`

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/MRBCVVrLM5n/a4cf1197-0c36-4f20-a8d6-2a0f4d17c9af?view=7d3bd44b-cfb9-4e77-a81b-e065d12bd460&t=attributes
- Added FAQ deprecation

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop FAQ in master, Save and check frontend.
- Switch to next-release, refresh the page and you will get validation issue.
- Switch to `faq-validation-issue-when-switch-to-master-to-next-release` issue will resolved.

### Checklist:
